README.rst

```diff
@@ -91,17 +91,14 @@ Settings
 Installation
```

Comment:
Need to update `docs` as well: https://github.com/JonasKs/django-guid/blob/dashed-guid-fix/docs/install.rst

Reply:
Done